docs/swagger/docs.go

```diff
@@ -796,6 +796,9 @@ var doc = `{
                 },
                 "schema-registry": {
                     "type": "string"
+                },
```

Comment:
cames from `make check`?

seems we can ignore it or file another pr

Reply:
d28f43f